modules/checkout/src/Plugin/Commerce/CheckoutPane/OrderSummary.php

```diff
@@ -42,6 +49,13 @@ public function buildConfigurationSummary() {
   public function buildConfigurationForm(array $form, FormStateInterface $form_state) {
     $form = parent::buildConfigurationForm($form, $form_state);
 
+    $form['use_views'] = [
+      '#type' => 'checkbox',
+      '#title' => $this->t('Use a View to display order summary'),
+      '#description' => $this->t('This setting allows you to use a View to override the template output of the order summary'),
```

Comment:
Bikeshed description.

Reply:
"Overrides the checkout order summary template with the output of a View."
Unsure if View should be capitalized, but the wording makes sense to me.